src/app/components/attestation/attestation-change-request.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { ThemeProvider } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardHeader from '@material-ui/core/CardHeader';
+import Typography from '@material-ui/core/Typography';
```

Comment:
I've been importing all of the elements from the same package using a single import. I don't know if it's better, but there are fewer characters, so maybe?
```suggestion
import  {
Button,
Card,
CardActions,
CardContent,
CardHeader,
Typography,
} from '@material-ui/core/Typography';
```